soc/renesas/ra/ra2/Kconfig.defconfig

```diff
@@ -12,4 +12,7 @@ config NUM_IRQS
 config LPM
 	default y
 
+config PINCTRL
+	default y
```

Comment:
-1, drivers need to select

Reply:
Could you explain? PINCTRL is a subsystem, not a driver.